### DISCUSSION TITLE
Shape of the reception wave

### DISCUSSION THREAD

**Discussion by ja-Bertolin** [AlexandreRouma/SDRPlusPlus#551] [General]
Hi

when receiving, the wave shape is quite “sparky” and not smooth, is there any way to soften it?
<img width="1303" alt="Screenshot 2021-12-16 at 23 07 30" src="https://user-images.githubusercontent.com/87009179/146456102-31794a4d-c682-401e-bd7b-7dec9ff0c2fe.png">
Thanks

**Comment by the-gobleo-tinkerer** [comment]
By "soften" do you mean time-averaging the spectrum? I don't think SDR++ has spectrum averaging at the moment, but you could open an Issue to request this feature.

**Reply by ja-Bertolin** [reply]
Yes indeed, apologies for not being accurate in my request. I will follow your recommendation, thanks.
